bitnami/thanos/values.yaml

```
@@ -1259,6 +1259,28 @@ storegateway:
   ##
   # existingConfigmap:
 
+  ## Thanos Store GRPC TLS parameters
+  ## to configure --grpc-server-tls-cert, --grpc-server-tls-key, --grpc-server-tls-client-ca
+  ## ref: https://github.com/thanos-io/thanos/blob/master/docs/components/store.md#flags
+  grpcTLS:
+    # TLS server side
+    server:
+      # enable TLS for GRPC server
+      secure: false
```

I'd use a different structure, sth like this:
```suggestion
  ## Thanos Store GRPC server configuration
  ##
  grpc:
    ## TLS configuration
    ##
    tls:
      ## Enable TLS encryption in the GRPC server
      ##
      enabled: false
```
What do you think?